dropwizard-jetty/src/main/java/io/dropwizard/jetty/setup/ServletEnvironment.java

```diff
@@ -130,26 +131,45 @@ public void setProtectedTargets(String... targets) {
     /**
      * Sets the base resource for this context.
      *
-     * @param baseResource The resource used as the base for all static content
-     *                     of this context.
+     * @param baseResource The resource to be used as the base for all static content of this context.
      */
     public void setBaseResource(Resource baseResource) {
         handler.setBaseResource(baseResource);
     }
 
     /**
-     * Sets the base resource for this context.
+     * Sets the base resources for this context.
      *
-     * @param resourceBase A string representing the base resource for the
-     *                     context. Any string accepted by Resource.newResource(String)
-     *                     may be passed and the call is equivalent to
-     *                     {@link #setBaseResource(Resource)}}
+     * @param baseResources The list of resources to be used as the base for all static
+     *                      content of this context.
+     */
+    public void setBaseResource(Resource... baseResources) {
+        handler.setBaseResource(new ResourceCollection(baseResources));
+    }
+
+    /**
+     * Sets the base resource for this context.
+     * @param resourceBase A string representing the base resource for the context. Any
+     *                     string accepted by Resource.newResource(String) may be passed
+     *                     and the call is equivalent to {@link #setBaseResource(Resource)}}
      */
     public void setResourceBase(String resourceBase) {
         handler.setResourceBase(resourceBase);
     }
 
     /**
+     * Sets the base resources for this context.
+     *
+     * @param resources A list of strings representing the base resources to serve static
+     *                  content for the context. Any string accepted by Resource.newResource(String)
+     *                  may be passed and the call is equivalent to {@link #setBaseResource(Resource...)}}
+     */
+    public void setResourceBase(String... resources) {
```

Comment:
Addressed in https://github.com/dropwizard/dropwizard/pull/1683/commits/a6b55d1c1dc48646370290a0d1d6c1fa0a04f633